app/domain/authentication/installed_authenticators.rb

```diff
@@ -33,21 +33,18 @@ def configured_authenticators
           .push(::Authentication::Common.default_authenticator_name)
       end
 
-      def enabled_authenticators(env)
+      def enabled_authenticators
         # Enabling via environment overrides enabling via CLI
-        env_enabled_authenticators(env) || db_enabled_authenticators
+        authenticators = 
+          Rails.application.config.conjur_config.authenticators.uniq
+        authenticators.empty? ? db_enabled_authenticators : authenticators
```

Comment:
I rarely feel like the ternary operator reads well.  What do you think about something like:
```ruby
def enabled_authenticators
  Rails.application.config.conjur_config.authenticators.select { |e| !e.to_s.empty? } ||
  db_enabled_authenticators
end
```

Reply:
I think I prefer the ternary in this case. Having a negated `select` that calls several methods on the elements it's selecting hurts my head more than the ternary does. 😄

Reply:
Also note that we are hopefully going to tear out db_enabled_authenticators soon-ish so whatever we choose here is going to be a temporary inconvenience.

Reply:
I personally think the ternary is clear here, but here's a non-ternary alternative which does the same thing:

```rb
authenticators = 
  Rails.application.config.conjur_config.authenticators.uniq
return authenticators if authenticators

db_enabled_authenticators
```